dynamic_rest/serializers.py

```diff
@@ -449,7 +449,7 @@ def to_internal_value(self, data):
         value = super(WithDynamicSerializerMixin, self).to_internal_value(data)
         id_attr = getattr(self.Meta, 'update_lookup_field', 'id')
         request_method = getattr(
-            getattr(self.context.get('view'), 'request'),
+            getattr(self.context.get('view'), 'request', None),
```

Comment:
According to Ryo's comment, DRF will add request onto the context by default, so you should just do `getattr(self.context, 'request', None)` instead of reaching into the view